packages/jsii-pacmak/lib/targets/go/types/type-member.ts

```diff
@@ -99,14 +99,15 @@ export class GoProperty implements GoTypeMember {
     const { code } = context;
     const receiver = this.parent.name;
     const instanceArg = receiver.substring(0, 1).toLowerCase();
+    const resultVar = slugify('r', [instanceArg]);
 
     code.openBlock(
       `func (${instanceArg} *${receiver}) ${
         this.getter
-      }()${` ${this.returnType}`}`,
+      }()${` (${resultVar} ${this.returnType})`}`,
```

Comment:
Whats the value of using a named return value here instead of letting the function body decide what the return var name is? I don't see the need for the parent data structure to care. I find named return values to generally be confusing. When someone sees `return`, it looks a lot like returning nil unless you go back up. Not that anyone is looking at our generated code but still.

I'm not super opinionated here if you want to keep them, though if you do lets also convert the `StaticGet` implementation to do the same thing. In the future I'd say all of the generated functions with return values should do it if we choose to go that way.

Reply:
Yeah if we can avoid this quirky idiom I'd be in favor.

Reply:
I guess the go style guide advises to avoid naming return values unless there's > 1 of them, so I guess fine...

The idea here was to actually avoid duplicating the return type signature - which I thought made it nicely apparent that the jsii runtime library call was actually putting it's result there (and that was __the__ result). But two of you agreeing that this isn't quite readable in this way (at least not without explanation) is enough to convince me otherwise 😅